src/main/java/com/conveyal/datatools/common/utils/Scheduler.java

```diff
@@ -56,42 +65,48 @@ public static void initialize() {
     /**
      * Convenience method for scheduling one-off jobs.
      */
-    public static ScheduledJob scheduleJob (String id, MonitorableJob job, long delay, TimeUnit timeUnit) {
+    public static ScheduledJob scheduleJob (FeedSource feedSource, Runnable job, long delay, TimeUnit timeUnit) {
```

Comment:
Let's rename this to be `scheduleFeedSourceJob` since it's only compatible with feed sources.  Please also update the javadoc for the method.